dml_events.go

```diff
@@ -433,8 +423,12 @@ func appendEscapedString(buffer []byte, value string) []byte {
 	return append(buffer, '\'')
 }
 
-func appendEscapedBuffer(buffer, value []byte) []byte {
-	buffer = append(buffer, "_binary'"...)
+func appendEscapedBuffer(buffer, value []byte, binary bool) []byte {
+	if binary {
+		buffer = append(buffer, "_binary"...)
+	}
+
+	buffer = append(buffer, '\'')
```

Comment:
I'm definitely stretching my ability to review this, but why do we need this extra ' here?

Reply:
This was in there previously as `_binary'`, as we still need to quote the data. It is just split now depending if you want the binary encoding or not.